telegram/_bot.py

```diff
@@ -1078,18 +1070,12 @@ async def send_audio(
         data: JSONDict = {
             "chat_id": chat_id,
             "audio": self._parse_file_input(audio, Audio, filename=filename),
+            "duration": duration,
+            "performer": performer,
+            "title": title,
+            "thumb": self._parse_file_input(thumb, attach=True),  # type: ignore[arg-type]
```

Comment:
Instead of the `type: ignore` we can probably improve the type annotation of `_parse_file_input` (maybe with `@override`) or just do something like `"thumb" = self._pfi(thumb) if thumb else None` - or in fact keep the if-clause as is :D